scripts/pmgr_check.py

```diff
@@ -0,0 +1,189 @@
+        the key from a pmgr configuration data dictionary
+
+    Returns
+    -------
+    str
+        a fully qualified EPICS PV
+    """
+    if 'FLD_' in key:
+        suffix = key.removeprefix('FLD')
+    elif 'PV_' in key:
+        suffix = key.removeprefix('PV')
+    else:
+        logger.debug(f'Unrecognized key provided: {key}')
+        return
+
+    # general string fixing... ew
+    suffix_parts = suffix.split("__")
+    new_suffix_list = [":".join(substr.split('_')) for substr in suffix_parts]
+    suffix = '_'.join(new_suffix_list)
+    suffix = ".".join(suffix.rsplit(":", 1))
+    pv = prefix + suffix
+    return pv
+
+
+def get_cfg_data(hutch: str, config_name: str) -> Dict[str, Any]:
+    """
+    Get pmgr config data corresponding to ``config_name`` and ``hutch``
+
+    Parameters
+    ----------
+    hutch : str
+        the hutch name, e.g. 'cxi'
+    config_name : str
+        the pmgr config name, e.g. 'KB1 DS SLIT LEF'
+
+    Returns
+    -------
+    Dict[str, Any]
+        The configuration values dictionary
+    """
+    pm = pmgrAPI.pmgrAPI('ims_motor', hutch.lower())
+    cfg_data = pm.get_config_values(config_name)
+
+    return cfg_data
+
+
+def create_atef_check(config_name: str, cfg_data: Dict[str, Any], prefix: str) -> PVConfiguration:
+    """
+    Construct the full atef checkout.  Simply creates an Equals comparison for each
+    value in the pmgr configuration, and groups it in a PVConfiguration
+
+    Parameters
+    ----------
+    config_name : str
+        the pmgr config name, e.g. 'KB1 DS SLIT LEF'
+    cfg_data : Dict[str, Any]
+        the configuration values dictionary, as returned from `get_cfg_data`
+    prefix : str
+        the EPICS Prefix
+
+    Returns
+    -------
+    PVConfiguration
+        The completed atef checkout
+    """
+    pv_config = PVConfiguration(name=f'check motor config: {config_name}',
+                                description='Configuration pulled from pmgr')
+
+    for key, value in cfg_data.items():
+        pv = get_pv(prefix, key)
+        if pv is None:
+            continue
+
+        comp = Equals(name=f'check for {pv}', description=f'Checking {pv} == {value}',
+                      value=value or 0)
+
+        # would need to handle first-time additions
+        pv_config.by_pv[pv] = [comp]
+
+    return pv_config
+
+
+def _create_arg_parser() -> argparse.ArgumentParser:
+    """Create the argparser."""
+    parser = argparse.ArgumentParser(
+        description=DESCRIPTION, formatter_class=argparse.RawTextHelpFormatter
+    )
+
+    parser.add_argument(
+        "hutch",
+        type=str,
+        help="name of hutch, e.g. 'cxi'",
+    )
+
+    parser.add_argument(
+        "pmgr_name",
+        type=str,
+        help="name of stored pmgr configuration, case and whitespace sensitive."
+             "e.g. 'KB1 DS SLIT LEF'",
+    )
+
+    parser.add_argument(
+        "prefix",
+        type=str,
+        help="EPICS PV Prefix, e.g. 'CXI:KB1:MMS:13'",
+    )
```

Comment:
There might be some additional utility in:

- Getting the values using the currently selected configuration for the prefix
- Allowing multiple prefix/config name pairs in one call

Not sure how much should be in scope for the script vs "import me and do the special thing you want"

Reply:
I wanted to avoid replicating functionality found in the pmgr cli utilities, so I think I lean toward not implementing suggestion 1 here.  

Suggestion 2 might be a smidge confusing in this script, but since we're lacking a good way to combine checkouts via the gui it might be necessary.  I'll add #2